File: Programacao/1010.c
#include<stdio.h>

int main(){
	
	int i,y,z;
	float total=0, x;
	
	for(i=0;i<2;i++){
	scanf("%d %d", &y, &z);
	scanf("%f", &x);
	
	total=total+z*x;
	}
	
	printf("VALOR A PAGAR: R$ %.2f\n", total);
	
	return 0;
	
}

File: Java/1010.java
import java.util.Scanner;

class Main{

    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        int x, y, a, b;
        double z, total, c;

        x = entrada.nextInt();
        y = entrada.nextInt();
        z = entrada.nextDouble();
        a = entrada.nextInt();
        b = entrada.nextInt();
        c = entrada.nextDouble();

        total = (y * z) + (b * c);
        
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
    }
}